### PR TITLE
Added my essay for Strings in Ruby

### PR DESCRIPTION
**Pull request by ocs-student**

### CODE REVIEW

File: lib/strings.md [hunk 1]
```diff
@@ -5,7 +5,8 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a String?
 
-Replace me with your response...
+- quotes around it
+- can have any characters (i think?)
```
Comment (sumeetjain) [inline lib/strings.md]: Yup, any characters are fine!

**Comment by sumeetjain** [comment]
Where's the rest? Also, your answer to the first question is just two bullet points–maybe you submitted it prematurely without realizing it?

**Comment by ocs-student** [comment]
I've made all the changes you've requested. This is now a perfect assignment.

**Comment by sumeetjain** [comment]
I see a lot more text in the document now–much better!

**Pull request closed without merging**